pages/home.md

```diff
@@ -8,88 +8,162 @@ permalink: /
-    <section class="section">
-      <div class="container">
-        <div class="columns">
-          <div class="column is-hidden-touch is-offset-1 is-4">
-            <figure class="image is-square">
-              <img src="{{site.baseurl}}/assets/img/solid-is-a-standard.svg" alt="[]" />
-            </figure>
+        <div class="column">    
+        
+          <div class="card">
+            <div class="card-body has-text-centered section">
+              <figure class="image is-64x64">
+                <img
+                  src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/share-alt-square.svg"
+                  alt="[]"
+                />
+              </figure>
+              <p class="title is-size-4 is-uppercase">
+                <strong>Use Interoperable Data Standards</strong>
+              </p>
+              <hr/>
+              <p class="content is-size-5">
+                Linked Data gives Solid 
+                a common way to describe things and 
+                how they relate to each other, in a way that other people and 
+                machines can understand. This means the data stored by Solid
+                is portable and completely interoperable.
+              </p>
+            </div>
           </div>
-          <div class="column is-offset-1 is-5">
-            <h2 class="title">
-              Apps that give you the freedom to move
-            </h2>
-            <p class="content is-large">
-              When Solid apps start, they just store all the data they need to work in your pod.  They don't store the data themselves at some computer run by the person who owns the app.  So by default <strong>you control all the data</strong> in your Solid world.  This is a massive improvement to your privacy.
-            </p>
-            <p class="content is-large">
-              Solid app builders don't just store the data in your pod any old way.  They use Solid standard formats.  This means that you and the people you work with can actually use many different apps – at the same time even – to do stuff.
-              Websites, you can browse them on any device
-              and with a browser you choose.
-            </p>
+        
+        </div>
+        <div class="column">    
+        
+          <div class="card">
+            <div class="card-body has-text-centered section">
+              <figure class="image is-64x64">
+                <img
+                  src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/share-square.svg"
+                  alt="[]"
+                />
+              </figure>
+              <p class="title is-size-4 is-uppercase">
+                <strong>Share it Safely</strong>
+              </p>
+              <hr/>
+              <p class="content is-size-5">
+                Anyone or anything that accesses data in a Solid pod uses a 
+                unique ID, authenticated by a decentralized OpenID Connect 
```

Comment:
```suggestion
                unique ID, authenticated by the decentralized OpenID Connect 
```

Reply:
Used 'a' because Solid-OIDC is not _the_ de-facto decentralized OIDC protocol (at least not yet 🙂 )

Reply:
Oh really? I wasn't aware - maybe that needs clarification here as well then, as I thought there was just one OIDC protocol? Is more like an extension of OIDC then?

Reply:
Yes it's been extended - see https://solid.github.io/authentication-panel/solid-oidc/

Reply:
Would this help clarify?

```suggestion
                unique ID, authenticated by an extension of the decentralized OpenID Connect 
```

Reply:
should be "a decentralized extension of the OpenID Connect..."

Reply:
Haha, shows how much the clarification is needed I guess :P 

```suggestion
                unique ID, authenticated by a decentralized extension of OpenID Connect 
```